rust/parser/typeql.pest

```diff
@@ -5,17 +5,15 @@
  */
 
 eof_query = { SOI ~ query ~ EOI }
+eof_query_prefix = { SOI ~ query }
 eof_label = { SOI ~ label ~ EOI }
 eof_definition_function = { SOI ~ definition_function ~ EOI }
 eof_definition_struct = { SOI ~ definition_struct ~ EOI }
 
-// WARN: do not include in the autogenerated tests
-eof_query_prefix_partial = { SOI ~ query ~ any_partial ~ EOI }
```

Comment:
Interestingly, we don't actually need this any_partial, we can just drop the EOI.

Reply:
Where do we use the partial parsing in the first place?

Reply:
In console - we either get a user writing a query interactively, in which we wait to get a double newline before submitting (here we wouldn't need the prefix parsing). However, we sometimes get copy-pasted scripts:
```
transaction write test
match ...;
insert ...;

commit
```
Where we don't want the commit to be parsed as part of the input